SpatialGDK/Source/SpatialGDK/Private/SpatialGDKSettings.cpp

```diff
@@ -29,6 +29,7 @@ USpatialGDKSettings::USpatialGDKSettings(const FObjectInitializer& ObjectInitial
 	, bUseFrameTimeAsLoad(false)
 	, bCheckRPCOrder(false)
 	, bBatchSpatialPositionUpdates(true)
+	, bEnableServerQBI(bUsingQBI)
```

Comment:
Do you want it enabled by default?

Reply:
Yes we do.

Reply:
Yes, this would be the default. The optimisation (single-server mode), would be to switch it off.